Gruntfile.js

```diff
@@ -28,5 +29,6 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-exec');
   grunt.loadNpmTasks('grunt-mocha-test');
   grunt.registerTask('default', ['jshint', 'mochaTest']);
-  grunt.registerTask('fmt', ['exec']);
+  grunt.registerTask('fmt', ['exec:jsfmtLib', 'exec:jsfmtTests', 'exec:jsfmtGrunt', 'exec:jsfmtExamples']);
+  grunt.registerTask('verify', ['exec:verifyNoChanges']);
```

Comment:
Is `exec:verifyNoChanges` built-in somewhere? Could you add a comment? I couldn't find a reference in the source / Google.